framework/src/executioners/Transient.C

```diff
@@ -741,6 +708,33 @@ Transient::getTimeStepperName()
     return std::string();
 }
 
+std::string
+Transient::getTimeIntegratorName()
+{
```

Comment:
I like to have this on screen output. Should we make this a function in `TimeIntegrator::type()` ? In that way, we can just return `_ti->type()` here. We actually had a function `Transient::getTimeScheme()` returns `Moose::TimeIntegratorType`. If we have a stringfy function, we can use that function here as well. The reason I am asking is that there could be more time integrators.

Reply:
can do that

Reply:
I m not sure this well help actually but made the change. it s consistent with other enums anyway

Reply:
nevermind it works with type().

stringify would not work for your new types (outside of MOOSE) of time integrators